fastplotlib/layouts/_base.py

```diff
@@ -12,14 +12,16 @@
 from ..graphics._base import Graphic, GraphicCollection
 from ..graphics.selectors._base_selector import BaseSelector
 
+from ..utils import GraphicMethods
```

Comment:
let's make a new module specifically for this, like `_add_graphic_mixin`

Reply:
renamed the file to above, but I just put the new module in utils, can move somewhere else if you prefer

Reply:
Make the new module here in layouts